hack/ipclaim-template.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: ipam.cluster.x-k8s.io/v1alpha1
```

Comment:
Can you remove this file? this was used for CI but is not used anymore at all :-)

Reply:
Removed in the latest patch.